go.mod

```diff
@@ -95,8 +93,8 @@ require (
 	github.com/russellhaering/gosaml2 v0.8.1
 	github.com/russellhaering/goxmldsig v1.2.0
 	github.com/schollz/progressbar/v3 v3.11.0
-	github.com/sethvargo/go-diceware v0.2.1
-	github.com/sirupsen/logrus v1.8.1
+	github.com/sethvargo/go-diceware v0.3.0
+	github.com/sirupsen/logrus v1.9.0
```

Comment:
This may not have the intended effect, since we have a replace directive to use a fork down below.

Reply:
Ack. I've added a comment so it's easier to spot in the future.

I also tried to "downgrade" version numbers closer to where the forks seem to be, but for some reason `go mod tidy` bumps them all back up.